pypopquiz/sources.py

```diff
@@ -29,9 +30,17 @@ def get_source(source_data: Dict[str, Any], output_dir: Path, input_dir: Path, w
 
     if source_type == "youtube":
         ppq.io.log("Downloading video '{:s}' from Youtube...".format(source_url))
-        video = YouTube("https://www.youtube.com/watch?v={:s}".format(source_url))
-        video = video.streams.filter(subtype=source_data["format"]).first()
-        video.download(output_path=str(output_dir / ppq.io.SOURCES_BASE_FOLDER), filename=source_url)
+
+        if downloader == 'pytube':
```

Comment:
We could maybe also put this in a try-catch block and don't use a command line argument to select it. But this also has advantages, you can now even use another downloader if pytube does work. So let's keep it like this :-)

Reply:
I understand what you mean. I wanted it to be explicit, especially since we don't officially have youtube-dl as dependency (we could add it to setup.py).

I will add a different try-catch though, to catch the cases where the youtube-dl exe is not found.